metrics/network/network-metrics-iperf3.sh

```diff
@@ -75,10 +94,29 @@ function iperf3_bidirectional_bandwidth_client_server {
 	local client_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -c ${server_address} -d -t ${time}"
 	start_client "$extra_args" "$client_name" "$image" "$client_command" > "$result"
 
-	local total_bidirectional_client_bandwidth=$(cat $result | tail -n 3 | head -1 | awk '{print $(NF-2), $(NF-1)}')
-	local total_bidirectional_server_bandwidth=$(cat $result | tail -n 4 | head -1 | awk '{print $(NF-3), $(NF-2)}')
-	echo "Network bidirectional bandwidth (client to server) is : $total_bidirectional_client_bandwidth"
-	echo "Network bidirectional bandwidth (server to client) is : $total_bidirectional_server_bandwidth"
+	local client_result=$(grep -m1 -E '\breceiver\b' $result)
+	local server_result=$(grep -m1 -E '\bsender\b'   $result)
+	local -a client_results
+	read -a client_results <<< ${client_result}
+	read -a server_results <<< ${server_result}
+	local total_bidirectional_client_bandwidth=${client_results[6]}
+	local total_bidirectional_client_bandwidth_units=${client_results[7]}
+	local total_bidirectional_server_bandwidth=${server_results[6]}
+	local total_bidirectional_server_bandwidth_units=${server_results[7]}
+	echo "Network bidirectional bandwidth (client to server) is :" \
+		"$total_bidirectional_client_bandwidth" \
+		"$total_bidirectional_client_bandwidth_units"
+	echo "Network bidirectional bandwidth (server to client) is :" \
+		"$total_bidirectional_server_bandwidth" \
+		"$total_bidirectional_server_bandwidth_units"
+
+	save_results "${test_name}" "network bidir bw client to server" \
```

Comment:
Ah, and here I see what you did with the multi-save of results into a single file... I think that is OK, but you are 'pushing the boundaries' here of what we have in place, so, some notes:
- use of the `Args` field is OK. That was not its original intention (it was to store the args the test had used...), but, it is not really being used or useful for that right now - so, I'd call that acceptable...
- placing multiple mixed results in a single results file will almost definitely break https://github.com/clearcontainers/tests/tree/master/cmd/checkmetrics parsing of those files... I think we can do an interim fix (filter the CSV files with `grep` for instance right now to get the results we need to check), and then add `Args` filtering to `checkmetrics`.

So, not the end of the world, and a pragmatic step forwards in holding mixed and varied results.